quickwit/quickwit-cli/src/index.rs

```diff
@@ -1098,6 +1103,16 @@ pub async fn delete_index_cli(args: DeleteIndexArgs) -> anyhow::Result<()> {
     debug!(args=?args, "delete-index");
     quickwit_telemetry::send_telemetry_event(TelemetryEvent::Delete).await;
 
+    if args.safe_mode {
+        let prompt = format!(
+            "This operation will delete the index `{}`. Do you want to proceed?",
+            args.index_id
+        );
+        if !prompt_confirmation(&prompt, false) {
+            return Ok(());
+        }
+    }
+
```

Comment:
Could we add a unit test for this? I think you can probably copy what we have done with `--dry-run`

Reply:
I tryed, but I clashed with [prompt_confirmation](https://github.com/quickwit-oss/quickwit/blob/main/quickwit/quickwit-cli/src/lib.rs#L163), because I have no idea how to "bypass" it. I though about putting something that conditionally *NOP*ped the function `prompt_confirmation`, but I didn't know how to do that. Maybe I missed the point where dry-run was tested, that's the most probable explanation.

Reply:
could you point me to the right code to replicate @ChillFish8, please?

Reply:
Ah yes, that is awkward, in that case we may have to skip the unit test although luckily it's simple logic. Thoughts @fmassot ?

Reply:
I think we should remain consistent with other prompt confirmations. This would means that by default, we have a prompt, and we can use `--yes` args to make the tests pass.

Reply:
Not a problem for me, but wouldn't this change existing behavior? If there are automated pipelines, with this update that seems minor, they would break. Again, not a problem for me, just want to make sure. :) You for sure know your user base better than me!

Reply:
I'm more in favor of consistency on all commands. just pinging @guilload to double check.

Reply:
I agree with you, @fmassot.

Reply:
@druzn3k can you update the PR to use a `--yes` flag?

Reply:
Yes, tomorrow I will 👍🏻

Reply:
Sorry for the latency, life got in the way (positively, nothing bad luckily) and still I didn't have time to apply the required changes. I promise that I will do it as soon as I can!